[PATCH] fix the start-statd

Upstream-Status: Pending

1. add /bin/ to PATH, since systemctl is installed under /bin/
2. redirect error to /dev/null

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 utils/statd/start-statd |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/statd/start-statd b/utils/statd/start-statd
index cde3583..098ce70 100644
--- a/utils/statd/start-statd
+++ b/utils/statd/start-statd
@@ -4,8 +4,8 @@
 # /var/run/rpc.statd.pid).
 # It should run statd with whatever flags are apropriate for this
 # site.
-PATH=/sbin:/usr/sbin
-if systemctl start statd.service
+PATH=/sbin:/usr/sbin:/bin
+if systemctl start statd.service 2>/dev/null
 then :
 else
     exec rpc.statd --no-notify
-- 
1.7.10.4

